test/parallel/test-http-set-timeout-server.js

```diff
@@ -152,11 +152,12 @@ test(function serverResponseTimeoutWithPipeline(cb) {
 });
 
 test(function idleTimeout(cb) {
-  const server = http.createServer(common.mustCall((req, res) => {
+  // Do not wrap the callback in common.mustCall(). It might not be invoked.
+  const server = http.createServer((req, res) => {
```

Comment:
If it *is* invoked, we want to make sure `timeout` event does not fire on `req` and `res` (I guess?).